app/controllers/application_controller.rb

```diff
@@ -63,7 +63,8 @@ def create_default_room(user)
     return unless user.rooms.count <= 0
     return unless PermissionsChecker.new(permission_names: 'CreateRoom', user_id: user.id, current_user: user, current_provider:).call
 
-    Room.create(name: "#{user.name}'s Room", user_id: user.id)
+    locale = I18n.default_locale unless valid_locale?(user.language.to_sym)
+    Room.create(name: t('room.new_room_name', username: user.name, locale:), user_id: user.id)
```

Comment:
I believe that we can assume the user's language is always valid - why the extra check here?

Reply:
I added the check because the tests were failing due to the user's language being set to "language". I don't know whether or not it's possible for a user's language to exist without a corresponding locale file, but if not I can also change the tests to pass a valid language

Reply:
Yeah I would update the tests instead